frontends/ol-util/assets/breakpoint.scss

```diff
@@ -1,26 +1,74 @@
-@mixin breakpoint($name) {
-  @if $name == desktop {
```

Comment:
The diff here is slightly confusing. This is exactly the file that used to be called `infinite-corridor/src/scss/breakpoints.scss`.

In this PR:
- `infinite-corridor/src/scss/breakpoints.scss` became new `ol-util/src/scss/breakpoint.scss`
- old `ol-util/src/scss/breakpoint.scss`, which was not used in new codebase, became this file.